components/brave_rewards/resources/page/components/settings.tsx

```diff
@@ -55,6 +56,10 @@ export function Settings () {
     return false
   }
 
+  React.useEffect(() => {
```

Comment:
Can this go into the big `useEffect` call that pulls in all of the other data?

Reply:
Fixed via d1376ae4b6a10a4baa57563a2fb00edbaf6ae482